oidc-samples/org.wso2.identity.sdk.oidc/org.wso2.identity.sdk.oidc/obj/Release/netstandard2.0/org.wso2.identity.sdk.oidc.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,26 @@
+C:\Users\wso2\source\repos\Project - IAM\org.wso2.identity.sdk.oidc\org.wso2.identity.sdk.oidc\bin\Release\netstandard2.0\org.wso2.identity.sdk.oidc.deps.json
```

Comment:
We cannot have absolute paths. Shall we try to move them to relative paths?

Reply:
Remove those files.